src/App.tsx

```diff
@@ -26,7 +26,7 @@ import { MessageContextProvider } from './contexts/MessageContext';
 import { SourceTargetContextProvider } from './contexts/SourceTargetContextProvider';
 import { TransactionContextProvider } from './contexts/TransactionContext';
 import { useApiConnection } from './hooks/useApiConnection';
-import Main from './screens/Main';
+import Main from './components/Main';
```

Comment:
I would not move the Main page inside the components. I normally prefer a distinguish between reusable components and page-components.